habitat/utils/gym_definitions.py

```diff
@@ -38,12 +38,31 @@ def _get_env_name(cfg: Config) -> Optional[str]:
     if (
         "GYM" in cfg
         and "AUTO_NAME" in cfg["GYM"]
+        and cfg["GYM"]["CLASS_NAME"] is not None
         and len(cfg["GYM"]["CLASS_NAME"]) > 1
     ):
         return cfg["GYM"]["CLASS_NAME"]
     return None
 
 
+def make_gym_from_config(config: Config) -> HabRenderWrapper:
+    """
+    From a habitat-lab or habitat-baseline config, create the associated gym environment.
+    """
+    if "TASK_CONFIG" in config:
+        config = config.TASK_CONFIG
```

Comment:
The reason this line is here is so that make_gym_from_config can take as argument either a habitat-lab or habitat-baseline configuration to create the environment. When the two are better separated, we will be able to enforce that make_gym_from_config takes only habitat-lab configs.